content/docs/getting-started.md

```diff
@@ -112,10 +112,16 @@ These are a combination of `<transport address>/ipfs/<hash-of-public-key>`.
 
 Now, you should be able to get objects from the network. Try:
 
+Linux:
 ```
 ipfs cat /ipfs/QmW2WQi7j6c7UgJTarActp7tDNikE4B2qXtFCfLPdsgaTQ/cat.jpg >cat.jpg
 open cat.jpg
 ```
+Windows:
+```
+ipfs cat /ipfs/QmW2WQi7j6c7UgJTarActp7tDNikE4B2qXtFCfLPdsgaTQ/cat.jpg >cat.jpg
+start cat.jpg
```

Comment:
`start` can also be used to run binaries. I'm hesitant to tell users to do this (won't be an issue in this case as windows cares about file extensions but still...).

Reply:
We should generalize the instructions and after the `cat` call just say "then open the file". Or we can replace the file with some ascii-art of a cat, and just do `cat` afterwards in macOS and `type` for windows.